src/main/java/hybrid/it/internship/library/exception/ExceptionController.java

```diff
@@ -0,0 +1,39 @@
+package hybrid.it.internship.library.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import javax.servlet.http.HttpServletRequest;
+
+@ControllerAdvice
+public class ExceptionController {
+
+    @ExceptionHandler(value = EntityNotFoundException.class)
+    @ResponseBody
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ExceptionResponse handleEntityNotFound(final EntityNotFoundException exception,
+                                                    final HttpServletRequest request) {
+        ExceptionResponse error = new ExceptionResponse();
+        error.setErrorMessage("Entity not found!");
```

Comment:
Set the error message from the EntityNotFoundException instead of "Entity not found!"